src/dart-2.md

```diff
@@ -43,8 +43,8 @@ and web development tools have changed.
   Instead, use the [new build system.][build system]
 * Tools related to web development have changed.
   * The new build system [replaces `pub build` and `pub serve`.][build_runner web]
-  * Dartium is no longer supported. Instead, use [dartdevc][] and Chrome.
-
+  * Dartium is no longer supported. 
```

Comment:
Maybe we can delete mentions of Dartium completely?

Reply:
This dart-2 page is explicitly about the historical changes which happened between Dart 1 and 2, so it still seems relevant (at least relative to the rest of the page). Perhaps we can explore retiring the page completely in the future.